packages/playground/public/info/static_website.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Static Website
```

Comment:
Can we come up with a better application name?

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/fa72219f-dbc1-4bc7-9d9e-d262864b9507)

Reply:
As per discussion with @AhmedHanafy725, This should be the application name